composer.json

```diff
@@ -36,7 +36,7 @@
         "nette/caching": "^3.1",
         "nette/robot-loader": "^3.4",
         "nette/utils": "^3.2",
-        "nikic/php-parser": "^4.10.4",
+        "nikic/php-parser": "dev-master as 4.10.4",
```

Comment:
4.10.5 just released https://github.com/nikic/PHP-Parser/releases/tag/v4.10.5

Reply:
Thanks :+1: 

I just got the notification too, I follow the releases.